etc/scripts/docenizer.py

```diff
@@ -7,12 +7,13 @@
 import sys
 import tarfile
 import urllib
-from urlparse import urljoin
+from urllib import request
+from urllib import parse
 
 try:
     from bs4 import BeautifulSoup
 except ImportError:
-    raise ImportError("Please install BeautifulSoup (apt-get install python-bs4 should do it)")
+    raise ImportError("Please install BeautifulSoup (apt-get install python-bs4 or pip install beautifulsoup4 should do it)")
```

Comment:
Thanks for updating this. We should probably do what we do in compiler-explorer/infra and set up a virtual environment, but this is of course :+1: for now